src/layouts/MainLayout.vue

```diff
@@ -13,14 +13,52 @@
       </q-toolbar>
 
       <q-tabs align="left">
-        <q-route-tab to="/page1" label="Page One" />
-        <q-route-tab to="/page2" label="Page Two" />
-        <q-route-tab to="/page3" label="Page Three" />
+        <q-route-tab :to="{ name: 'home' }" label="Home" />
+        <q-route-tab to="/page2" label="Corsi" />
+        <q-route-tab to="/page3" label="About" />
+        <q-route-tab to="/page4" label="❁" />
+        <q-route-tab to="/page5" label="Log In" />
```

Comment:
Se non hai nessuna pagina da collegare non mettere MAI cose finte o temporanee. Piuttosto togli la prop
In piu controlla la proprietà "disable".
Disabilita il componente così nn puoi cliccarlo e gli applica gia lo stile